pikvm_comm.py

```diff
@@ -344,3 +344,24 @@ def UploadImagePiKVM(
     x = requests.post(uri, headers=headers_login, verify=False)
     uri = f"https://{pikvm_ip}/api/msd/write_remote?url={img_url}"
     x = requests.post(uri, headers=headers_login, verify=False)
+
+
+@keyword("Copy Image PIKVM")
+def CopyImagePiKVM(
```

Comment:
@wiktormowinski It seems like this is almost identical to `UploadImagePiKVM` with a single change of removing `?url=` from the request.

Why not use another function to share the common code between these two?

Also, I would suggest to use better naming, so it's obvious that one kwd works on file, while the second one works on URL. Now you simply changed `Upload` to `Copy`. We still perform the upload, but simply from local path, not from the remote URL.

Better names could be something like: `Upload From File / Upload From URL`. Or even further, if these keywords are so similar, you can make it accept both URL and local path, and then adjust request accordingly. Either automatically based on the argument, or manually, by introducing second argument which switches between file and URL.